docs/data-ingestion/Proxmox.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Proxmox
+weight: 1
+sort: 1
+menu:
+  docs:
+    identifier: "proxmox"
+    parent: "data-ingestion"
+    weight: 1
+    # sort: 1
+aliases:
+  - /data-ingestion/proxmox.html
+  - /data-ingestion/Proxmox.html
```

Comment:
@tenmozes can we make http router in Hugo to be case insensetive?

Reply:
Hugo is case insensitve but GCS is not, so if we deploy our site somewhere else it will be case insensitive

Reply:
the best thing that we can do is to use lowercase